quickstarts/microsoft.digitaltwins/digitaltwins-with-function-time-series-database-connection/azuredeploy.parameters.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "digitalTwinsName": {
+            "value": "GEN-UNIQUE"
+        },
+        "eventHubsNamespaceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "eventHubName": {
+            "value": "GEN-UNIQUE"
+        },
+        "adxClusterName": {
+            "value": "GEN-UNIQUE"
+        },
+        "databaseName": {
+            "value": "GEN-UNIQUE"
+        },
+        "databaseTableName": {
+            "value": "GEN-UNIQUE"
+        },
+        "eventHubsNamespacePlan": {
```

Comment:
remove non-GEN values from here and use defaultValues in the template where possible

Reply:
Addressed